README.md

```diff
@@ -74,94 +74,7 @@ karpenter-5f97c944df-bm85s 1/1   Running 0        15m
-
-Here's the default `Provisioner` and `AWSNodeTemplate` manifest you'll deploy, take your time to explore the configurations of the default provisioner as it's trying to be as flexible as possible:
-
-```
-apiVersion: karpenter.sh/v1alpha5
-kind: Provisioner
-metadata:
-  name: default
-spec:
-  labels:
-    intent: apps
-  requirements:
-    - key: "karpenter.k8s.aws/instance-category"
-      operator: In
-      values: ["c", "m", "r", "i", "d"]
-    - key: "karpenter.k8s.aws/instance-cpu"
-      operator: In
-      values: ["4", "8", "16", "32", "48", "64"]
-    - key: karpenter.sh/capacity-type
-      operator: In
-      values: ["spot", "on-demand"]
-    - key: kubernetes.io/arch
-      operator: In
-      values: ["amd64", "arm64"]
-  kubeletConfiguration:
-    containerRuntime: containerd
-  limits:
-    resources:
-      cpu: 100000
-      memory: 5000Gi
-  consolidation:
-    enabled: true
-  ttlSecondsUntilExpired: 604800
-  providerRef:
-    name: default
----
-apiVersion: karpenter.k8s.aws/v1alpha1
-kind: AWSNodeTemplate
-metadata:
-  name: default
-spec:
-  subnetSelector:
-    karpenter.sh/discovery: "<<CLUSTER_NAME>>"
-  securityGroupSelector:
-    karpenter.sh/discovery: "<<CLUSTER_NAME>>"
-  instanceProfile: "<<KARPENTER_NODE_IAM_INSTANCE_PROFILE_NAME>>"
-  tags:
-    karpenter.sh/discovery: "<<CLUSTER_NAME>>"
-    intent: apps
-    project: karpenter-blueprints
-    KarpenterProvisionerName: "default"
-    NodeType: "default"
-    IntentLabel: "apps"
-```
-
-If you're using the Terraform template provided in this repo, go back to the root folder of this project and run the bellow commands to get the EKS cluster name and the IAM Role name for the Karpenter nodes. Otherwise, make sure you provide the proper values for the environment variables we'll use during this project. Run this to get the values from Terraform:
-
-```
-export CLUSTER_NAME=$(terraform -chdir="cluster/terraform" output -raw cluster_name)
-export KARPENTER_NODE_IAM_INSTANCE_PROFILE_NAME=$(terraform -chdir="cluster/terraform" output -raw node_instance_profile_name)
-```
-
-***NOTE***: If you're not using Terraform, you need to get those values manually. `CLUSTER_NAME` is the name of your EKS cluster (not the ARN), and `KARPENTER_NODE_IAM_INSTANCE_PROFILE_NAME` is the [instance profile](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_use_switch-role-ec2_instance-profiles.html#instance-profiles-manage-console), which is a way to pass a single IAM role to the EC2 instance launched by the Karpenter provisioner. Typically, the instance profile name is the same as the IAM role, but to avoid errors, go to the IAM Console and get the instance profile name assigned to the role (not the ARN).
```

Comment:
This is an important paragraph for those who are using an existing cluster, let's keep it.